server/routes/reporting/standard-report/definitions.js

```diff
@@ -405,6 +405,19 @@
           totals: true,
         },
       },
+      'manual-juror-report': {
+        title: 'Manually-created jurors report',
+        apiKey: 'ManuallyCreatedJurorsReport',
+        search: 'dateRange',
+        headings: [
+          'dateFrom',
+          'reportDate',
+          'dateTo',
+          'reportTime',
+          'totalManuallyCreatedJurors',
+          'courtName',
+        ],
+      },
```

Comment:
this report needs to be printed in landscape mode with: `landscape: true`

Reply:
Updated